src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -36,14 +38,22 @@
     public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
 
-    public static final String VALID_TRIPDAY_FRIDAY = "FRIDAY";
-    public static final String VALID_TRIPDAY_MONDAY = "MONDAY";
-    public static final String VALID_TRIPTIME_EVENING = "1800";
-    public static final String VALID_TRIPTIME_MORNING = "0830";
-    public static final String VALID_TRIPDAY_BOB = "THURSDAY";
-    public static final String VALID_TRIPTIME_BOB = "1930";
-    public static final String VALID_PRICE_AMY = "1.69";
-    public static final String VALID_PRICE_BOB = "6.9";
+    public static final DayOfWeek VALID_TRIPDAY_FRIDAY = DayOfWeek.FRIDAY;
+    public static final DayOfWeek VALID_TRIPDAY_MONDAY = DayOfWeek.MONDAY;
+    public static final LocalTime VALID_TRIPTIME_EVENING = LocalTime.of(18, 00);
+    public static final LocalTime VALID_TRIPTIME_MORNING = LocalTime.of(8, 30);
+    public static final DayOfWeek VALID_TRIPDAY_BOB = DayOfWeek.THURSDAY;
+    public static final LocalTime VALID_TRIPTIME_BOB = LocalTime.of(19, 30);
+    public static final double VALID_PRICE_AMY = 1.69;
+    public static final double VALID_PRICE_BOB = 6.9;
```

Comment:
should these be the actual values in the native types?
or should these be the strings, because i believe command test util is to test the input that the user puts in (in the form of a string). To test if it is correct or wrong input

Reply:
Thanks for the comment! To clarify, the VALID_[VARIABLE] constants are not used as replacements for user input but rather for the personbuilder and other such utility functions during testing. As for tests concerning user inputs, the constants declared below of the form [VARIABLE]_DESC_BOB, and as they are supposed to emulate user inputs, they are of the String datatype. Hope this addresses your comment!